### ISSUE TITLE
font donwloading issue

### ISSUE THREAD

**Issue by ghost** [chawyehsu/mactype-profile#1]
Can you please provide an easier download method for that font family?

**Comment by chawyehsu** [comment]
Hi @Gigobitto , I uploaded a copy of the font family to my OneDrive, and here is the access link: https://1drv.ms/f/s!Aq2u7CJXAQY6j4QfqqHkz_UTY6KdiQ . The typeface was designed in 2014, which is kinda old and it's specifically designed for Chinese usage, English is OK though, tell me if it works for you. Thank you for having interest in my MacTpye profile, feel free to give me feedback if you have any other question.

**Comment by ghost** [comment]
I haven't spotted any issues outside of some really strange font substitutions in Firefox. Stuff like Open file..., Save page as...., Print..., has the dots replace by asteriscs; this obviously doesn't only happen in menus, but in webpages too

**Comment by chawyehsu** [comment]
@Gigobitto I can't reproduce the issue you describe. Could you please provide some screenshots? Mine works fine.

![20190703214701](https://user-images.githubusercontent.com/5764917/60596764-3e979c00-9ddc-11e9-8cea-6e9c23dcb311.png)

Have you turned GDI on in the Firefox? If not, please try to turn it on, following MacType's [wiki](https://github.com/snowie2000/mactype/wiki/Mozilla-Firefox).

**Comment by ghost** [comment]
The asteriscs you have there after "Open file", "Save page as", "Print", etc. are meant to be dots

**Comment by chawyehsu** [comment]
Nope, they are not asteriscs, they are "⋯" (U+22EF Midline Horizontal Ellipsis), that's because the XHei WP font is designed for Chinese. And it's a right behavior for Chinese characters.

![20190703224910](https://user-images.githubusercontent.com/5764917/60601497-cf727580-9de4-11e9-8ed6-17ad0f3f90c3.png)

If you do care about the issue, I suggest you turning off the font substitutions.

**Comment by marcuslannister** [comment]
@chawyehsu https://1drv.ms/f/s!Aq2u7CJXAQY6j4QfqqHkz_UTY6KdiQ this download link is broken, could you please  upload again?

**Comment by chawyehsu** [comment]
It's still available, please try again. It was shared with no expired date.

**Comment by marcuslannister** [comment]
I use another network and find it is ok now, thanks.
